dev/tools/backport-pr.sh

```diff
@@ -30,7 +30,13 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
-MASTER=origin/master
+REMOTE=$(git config --get "branch.master.remote")
```

Comment:
There's the same issue here as I reported in https://github.com/coq/coq/issues/12314.  You'll never actually trigger the `if` branch without making this change
```suggestion
REMOTE=$(git config --get "branch.master.remote" || true)
```

Reply:
Thanks!

Reply:
@JasonGross Fixed in this tool and all the others.